tests/unit/synthesizer/test_ctgan.py

```diff
@@ -289,3 +291,61 @@ def test__validate_discrete_columns(self):
         ctgan = CTGAN(epochs=1)
         with pytest.raises(ValueError, match=r'Invalid columns found: {\'doesnt exist\'}'):
             ctgan.fit(data, discrete_columns)
+
+    def test__validate_null_data(self):
+        """Test `_validate_null_data` with pandas and numpy data.
+
+        Check the appropriate error is raised if null values are present in
+        continuous columns, both for numpy arrays and dataframes.
+
+        Setup:
+            - Create dataframe with a continuous column
+            - Create numpy array with same data
+            - Create dataframe with a discrete column
+            - Create numpy array with a discrete column
+
+        Input:
+            - train_data = 2-dimensional numpy array or a pandas.DataFrame
+            - discrete_columns = list of strings or integers
+
+        Output:
+            None
+
+        Side Effects:
+            - Raises error if a continuous column contains a null value.
+
+        Note:
+            - could create another function for numpy array
```

Comment:
minor: this is an old doc style we don't use anymore, you can just have the summary for new tests